lib/seahorn/BvOpSem2.cc

```diff
@@ -1810,11 +1810,15 @@ Expr Bv2OpSemContext::getConstantValue(const llvm::Constant &c) {
     if (GVO.hasValue()) {
       GenericValue gv = GVO.getValue();
       if (gv.AggregateVal.size() > 0) {
-        APInt aggBv = m_sem.agg(c.getType(), gv.AggregateVal, *this);
-        expr::mpz_class k = toMpz(aggBv);
-        return alu().si(k, aggBv.getBitWidth());
+        auto aggBvO = m_sem.agg(c.getType(), gv.AggregateVal, *this);
+        if (aggBvO.hasValue()) {
+          APInt aggBv = aggBvO.getValue();
```

Comment:
skip this, or make `aggBv` a reference. Otherwise there is an extra unnecessary copy.